### PR TITLE
Feature/codestyle upd

### PR DESCRIPTION
**Pull request by doxliydriw**
Updated ESLint configuration due to wrong linter check in subfolders.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@mui/base": "^5.0.0-beta.36",
```
Comment (DimaShubin) [inline package.json]: What did you add this for?

File: package.json [hunk 2]
```diff
@@ -15,6 +16,7 @@
     "axios": "^1.5.1",
     "babel-plugin-styled-components": "^2.1.4",
     "formik": "^2.4.5",
+    "jsdom": "^24.0.0",
```
Comment (DimaShubin) [inline package.json]: Do we really need it?

File: package.json [hunk 3]
```diff
@@ -33,16 +35,18 @@
     "sweetalert": "^2.1.2",
     "vite": "^5.0.12",
     "vite-tsconfig-paths": "^4.3.1",
+    "vitest": "^1.2.2",
```
Comment (DimaShubin) [inline package.json]: That is a topic for another task, am I right?

File: package.json [hunk 4]
```diff
@@ -33,16 +35,18 @@
...
     "web-vitals": "^2.1.4",
     "yup": "^1.3.2"
   },
   "scripts": {
     "start": "vite",
+    "test:unit": "vitest --root src/",
```
Comment (DimaShubin) [inline package.json]: Same here.

File: setupTest.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { expect } from 'vitest';
```
Comment (DimaShubin) [inline setupTest.js]: https://jasjamironec.atlassian.net/browse/TAL-100

File: src/App.test.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { describe, expect, test } from 'vitest';
```
Comment (DimaShubin) [inline src/App.test.jsx]: https://jasjamironec.atlassian.net/browse/TAL-100

File: src/components/ChatMap/ChatMap.jsx [hunk 7]
```diff
@@ -95,7 +95,9 @@ const ChatMap = ({ closeMap }) => {
         />
         <ShowCountry>
           <CountryName>{selectedCountry}</CountryName>
-          <button onClick={handleClick}>Join</button>
+          <button type="button" onClick={handleClick}>
```
Comment (DimaShubin) [inline src/components/ChatMap/ChatMap.jsx:98]: button by default has type==="button"

File: src/components/LoginForm/LoginForm.jsx [hunk 8]
```diff
@@ -1,25 +1,32 @@
+/* eslint-disable no-nested-ternary */
```
Comment (DimaShubin) [inline src/components/LoginForm/LoginForm.jsx]: What is the main idea of that PR? Add Eslint settings or customisation of login page?

File: src/components/Profile/Profile.jsx [hunk 9]
```diff
@@ -83,19 +83,22 @@ const Profile = () => {
...
         onSubmit={handleSubmit}
         validationSchema={validationFormikSchema}
       >
         {({
+          /* TODO: Check what below props are sued for and which handleSubmit should be called
+          in <Forms></Forms> //Kostas//
+           */
           values,
-          handleSubmit,
+          // handleSubmit,
```
Comment (DimaShubin) [inline src/components/Profile/Profile.jsx]: Don't leave any comments like this.

File: src/node_modules/.vitest/results.json
Comment (DimaShubin) [inline src/node_modules/.vitest/results.json:1]: Why do have the node_modules inside src/ folder?

File: src/routes/LoginRoute/LoginRoute.jsx [hunk 10]
```diff
@@ -1,8 +1,7 @@
-import React from 'react';
 import LoginForm from '@/components/LoginForm/LoginForm';
 
-const LoginRoute = () => {
+function LoginRoute() {
```
Comment (DimaShubin) [inline src/routes/LoginRoute/LoginRoute.jsx]: Change it back please.

File: vite.config.js [hunk 11]
```diff
@@ -22,6 +25,11 @@ export default defineConfig({
     }),
     viteTsconfigPaths(),
   ],
+  test: {
```
Comment (DimaShubin) [inline vite.config.js]: https://jasjamironec.atlassian.net/browse/TAL-100

**Review by DimaShubin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (doxliydriw) [reply package.json]: removed

File: package.json (hunk 3 above)
Comment (doxliydriw) [reply package.json]: removed

File: package.json (hunk 4 above)
Comment (doxliydriw) [reply package.json]: removed

File: src/components/LoginForm/LoginForm.jsx (hunk 8 above)
Comment (doxliydriw) [reply src/components/LoginForm/LoginForm.jsx]: removed

File: src/components/Profile/Profile.jsx (hunk 9 above)
Comment (doxliydriw) [reply src/components/Profile/Profile.jsx]: removed

File: vite.config.js (hunk 11 above)
Comment (doxliydriw) [reply vite.config.js]: removed

File: src/node_modules/.vitest/results.json
Comment (doxliydriw) [reply src/node_modules/.vitest/results.json:1]: removed. haven't noticed it appeared after vitest installation.

File: setupTest.js (hunk 5 above)
Comment (doxliydriw) [reply setupTest.js]: removed

File: src/routes/LoginRoute/LoginRoute.jsx (hunk 10 above)
Comment (doxliydriw) [reply src/routes/LoginRoute/LoginRoute.jsx]: removed.

**Review by DimaShubin — APPROVED** [review]

**Pull request merged**